scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -569,7 +639,9 @@ def _expm_multiply_interval(A, B, start=None, stop=None,
         raise ValueError('expected B to be like a matrix or a vector')
     u_d = 2**-53
     tol = u_d
-    mu = _trace(A) / float(n)
+    if traceA is None:  # TODO: we need a good number for m3
```

Comment:
Is this `TODO` a blocker?

Reply:
I don't think it is a blocker. This is purely an issue of performance.

The trace is a preconditioner improving the performance, `m3` influences the accuracy of the trace estimate.
I don't know any good heuristics to choose this parameter, thus the current value is more or less arbitrary.
But a slow result is better than none, and the user has the option to give the trace by hand to speed things up.

Reply:
@DerWeh It's not really a 'TODO' . Maybe add instead a comment something like that the default choice `m3=10` is quite arbitrary, where increasing it requires more calculations for the trace estimate, while decreasing it makes the estimate less accurate and thus slows down convergence or decreases the accuracy of the exponent calculation

Reply:
I would agree with adding a comment